code/modules/clothing/masks/gasmask.dm

```diff
@@ -147,17 +165,23 @@
 	flags_cover = MASKCOVERSEYES
 	resistance_flags = FLAMMABLE
 
+	species_restricted = FLAG_GROD
```

Comment:
Why do you keep adding one new line before every `species_restricted`?

Reply:
Not sure why Kapu did this